network-test-app/app.js

```diff
@@ -0,0 +1,57 @@
+const { Client } = require("pg");
+const express = require("express");
+const app = express()
+
+const port = process.env.PORT;
+
+app.get("/", async (request, response) => {
+    pgServiceCredentials = process.env.DATABASES_FOR_POSTGRESQL_CONNECTION
+    if(pgServiceCredentials != "" && pgServiceCredentials != undefined){
+        /* 
+            Postgres service credentials have been configured properly, 
+            continue with attempting to connect to service
+        */
+        try {
+            // Use env variables loaded from service binding to connect to our postgres instance
```

Comment:
Does a try/catch verify that the postgres connection works? Just want to test with a valid and invalid connection. Easy test might be to turn the public endpoints on and off.

Reply:
Verified that this try/catch verifies that the postgress connection works. When the public endpoint was turned off, the app returned a response saying it could not connect to the postgres instance